internal/staging/deploy.go

```diff
@@ -529,3 +559,17 @@ func (c *controller) isPVCsReady(listOpt *client.ListOptions) (bool, error) {
 
 	return true, nil
 }
+
+func (c *controller) checkAllReleasesDeployed(releases []*release.Release) (bool, bool) {
```

Comment:
can we define the return name?
when seeing only (bool, bool), it hard to understand what does it mean